src/style.css

```
@@ -0,0 +1,135 @@
+.product-card {
+  font-family: "Roboto", sans-serif;
+}
+
+.product-card__content,
+h2,
+h3 {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: "Roboto", sans-serif;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  display: flex;
+  justify-content: center;
+}
+/*--------------------Title-------------------*/
+h2 {
+  padding-top: 40px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.title__code {
+  height: 14px;
+  margin-top: 4px;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: #616070;
+  display: flex;
+}
+/*------------------review------------------------------*/
+.review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  
+}
+
+.review__stars {
+  display: flex;
+  padding-top: 15px;
+}
+
+.review__item {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star-active.svg);
+  background-position: 50%;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.review__item--last_color {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-position: 50%;
+  background-repeat: no-repeat;
+}
+
+.review__total {
+  padding-top: 15px;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: #060b35;
+}
+/*---------------------------Producte price-------------------------*/
+h3 {
+  height: 18px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.producte-price__left,
+.producte-price__right {
+  margin-top: 47px;
+}
```

```suggestion
}
```


add margin-bottom to block which is above